pkg/config/vpn-client.go

```diff
@@ -57,6 +64,34 @@ func GetVPNClientConfig() (VPNClient, error) {
 		return VPNClient{}, err
 	}
 	cfg.VPNClientIndex = -1
+
+	if cfg.IsHA {
+		if cfg.PodName == "" {
+			return VPNClient{}, fmt.Errorf("IS_HA is set to true but POD_NAME is not set")
+		}
+		if cfg.VPNServerIndex == "" {
+			return VPNClient{}, fmt.Errorf("IS_HA is set to true but VPN_SERVER_INDEX is not set")
+		}
+	}
+
+	if len(cfg.IPFamilies) > 2 {
+		return VPNClient{}, fmt.Errorf("IP_FAMILIES must not contain more than 2 elements")
+	}
```

Comment:
We should probably also check that the length is at least 1 as for example `PrimaryIPFamily()` assumes the length of at least 1.

Reply:
yes, we can do that. though it is technically impossible to set it to zero elements, because we have a default of `IPv4` that would kick in if the user provided an empty string or no value at all.